docs/source/overview/sensors/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _overview_sensors:
+
+Sensors
+=========
+
+In this section, we will overview the various sensor APIs provided by Isaac Lab.
+
+Every sensor in Isaac Lab inherits from the ``SensorBase`` abstract class that provides the core functionality inherent to all sensors, which is to provide access to "measurements" of the scene. These measurements can take many forms such as ray-casting results, camera rendered images, or even simply ground truth data queried directly from the simulation (such as poses). Whatever the data may be, we can think of the sensor as having a buffer that is periodically updated with measurements by querying the scene. This ``update_period`` is defined in "simulated" seconds, meaning that even if the flow of time in the simulation is dilated relative to the real world, the sensor will update at the appropriate rate. For example, a sensor with an update period of 1.0 second will update its data 10 times a second in a simulation running at a rate of 10 seconds / second. The ``SensorBase`` is also designed with vectorizability in mind, holding the buffers for all copies of the sensor across cloned environments.
```

Comment:
"10 seconds / second"?

Reply:
yes, i figured being explicit here with the units was best, in order to express that we were running at a rate of 10 seconds in the sim per second in reality.  I want to make sure people know that if they set the sensor update rate, that the rate in simulated time, not wall clock time